### PR TITLE
updated Links

### PR DESCRIPTION
**Pull request by trmccormick**
added https for links to the directory. Replaced My Access with portal. Adjusted some formatting and replaced some I tags with spans while I was in the files

### CODE REVIEW

File: illiad2015/FAQ.html [hunk 1]
```diff
@@ -267,43 +267,42 @@ <h3>Who do I contact if I have problems with or questions about ILLiad?</h3>
...
+						              <img src="https://lib.wvu.edu//images/2014/wvufacebook.png" alt="Facebook" title="Facebook"></img>
+						          </a>
+						        </li>
+        						<li>
+        						    <a class="socmedia" href="https://plus.google.com/100326194366145492876" rel="publisher" target="_blank">
+        						    	<img src="https://lib.wvu.edu//images/2014/wvugoogleplus.png" alt="Google+" title="Google+"</img>
+        							</a>
+        						</li>
+        						<li>
+        							<a class="socmedia" href="http://instagram.com/wvulibrarianlinda" target="_blank">
```
Comment (ddavisgraphics) [inline illiad2015/FAQ.html:289]: This may have to change I think its wvulibraries now, ask @jetapia

File: illiad2015/Logon.html [hunk 2]
```diff
@@ -87,17 +87,17 @@
...
 						<a class="library-header-logo" href="https://lib.wvu.edu/">WVU Libraries<i class="fa fa-university library-logo"></i></a>
 						<ul class="library-header-links">
-							<li><a href="https://lib.wvu.edu/"><i class="fa fa-university"></i>Libraries</a></li>
-		                    <li><a href="https://lib.wvu.edu//about/"><i class="fa fa-info-circle"></i>About Us</a></li>
-		            		<li><a href="https://lib.wvu.edu/collections/"><i class="fa fa-inbox"></i>Collections</a></li>
-		                    <li><a href="https://lib.wvu.edu//instruction"><i class="fa fa-graduation-cap"></i>Instruction</a></li>
+							  <li><a href="https://lib.wvu.edu/"><i class="fa fa-university"></i>Libraries</a></li>
+		            <li><a href="https://lib.wvu.edu//about/"><i class="fa fa-info-circle"></i>About Us</a></li>
+		            <li><a href="https://lib.wvu.edu/collections/"><i class="fa fa-inbox"></i>Collections</a></li>
+		            <li><a href="https://lib.wvu.edu//instruction"><i class="fa fa-graduation-cap"></i>Instruction</a></li>
```
Comment (ddavisgraphics) [inline illiad2015/Logon.html]: There is a double slash between instruction, why?

File: illiad2015/Logon.html [hunk 3]
```diff
@@ -225,42 +225,42 @@ <h4>Log on using MyID</h4>
...
+      						        <img src="https://lib.wvu.edu//images/2014/wvufacebook.png" alt="Facebook" title="Facebook"></img>
+      						    </a>
+						        </li>
+        						<li>
+        						    <a class="socmedia" href="https://plus.google.com/100326194366145492876" rel="publisher" target="_blank">
+        						    	<img src="https://lib.wvu.edu//images/2014/wvugoogleplus.png" alt="Google+" title="Google+"</img>
+        							</a>
+        						</li>
+        						<li>
+        							<a class="socmedia" href="http://instagram.com/wvulibrarianlinda" target="_blank">
```
Comment (ddavisgraphics) [inline illiad2015/Logon.html]: I think this should be wvulibraries again.

File: illiad2015/template_header_postVars.html [hunk 4]
```diff
@@ -81,17 +81,17 @@
...
-		    				<li><a href="https://lib.wvu.edu//services/"><i class="fa fa-cog"></i>Services</a></li>
-		                    <li><a href="https://lib.wvu.edu//services/ask/"><i class="fa fa-question-circle"></i>Get Help</a></li>
-		    				<li><a href="https://lib.wvu.edu//about/giving/"><i class="fa fa-heart"></i>Give</a></li>
+  							<li><a href="https://lib.wvu.edu/"><i class="fa fa-university"></i>Libraries</a></li>
+  		          <li><a href="https://lib.wvu.edu//about/"><i class="fa fa-info-circle"></i>About Us</a></li>
+  		          <li><a href="https://lib.wvu.edu/collections/"><i class="fa fa-inbox"></i>Collections</a></li>
+  		          <li><a href="https://lib.wvu.edu//instruction"><i class="fa fa-graduation-cap"></i>Instruction</a></li>
+  		    			<li><a href="https://lib.wvu.edu/about/news/"><i class="fa fa-rss"></i>News</a></li>
+  		    			<li><a href="https://lib.wvu.edu//services/"><i class="fa fa-cog"></i>Services</a></li>
+  		          <li><a href="https://lib.wvu.edu//services/ask/"><i class="fa fa-question-circle"></i>Get Help</a></li>
```
Comment (ddavisgraphics) [inline illiad2015/template_header_postVars.html]: Another odd link, please check this out.

**Review by ddavisgraphics — COMMENTED** [review]
There may need to be more changes other than what I mentioned.

File: illiad2015/Logon.html (hunk 2 above)
Comment (trmccormick) [reply illiad2015/Logon.html]: not sure I see that their is a double slash in about as well.

**Pull request merged**